### DISCUSSION TITLE
packaged appimages for aur

### DISCUSSION THREAD

**Discussion by Ckath** [Chatterino/chatterino2#2743] [Q&A] [answered]
created [chatterino2-appimage](https://aur.archlinux.org/packages/chatterino2-appimage/) and [chatterino2-nightly-appimage](https://aur.archlinux.org/packages/chatterino2-nightly-appimage/) according to the discussion in #2736 

added the same co-maintainers as the last one(including the ci), let me know if any one else needs to be added.

for the integration:
- the regular appimage needs `pkgver` and the `md5sums`(generated with makepkg -g) updated every release
- the nightly is done with a `pkgver` of the current date(can use anything really) and a static nightly-build url, bumping `pkgver` alone will be enough as md5sums are also skipped

if anything about these looks wrong(not very familiar with appimages myself) also feel free to let me know, this is how I saw most other `-appimage` packages done and it seemed logical to me.

**Comment by Ckath** [comment]
what I run manually now for updating these, not sure if they'd translate to ci and I'm too lazy to look at it tbh

chatterino2-appimage
```
sed -i "s/pkgver=.*/pkgver=$newver/" PKGBUILD
sed -i "s/md5sums.*/$(makepkg -g)/" PKGBUILD
makepkg --printsrcinfo > .SRCINFO
rm -rf Chatterino-x86_64.AppImage src
git add PKGBUILD .SRCINFO
git commit -m "update to $newver"
```
chatterino2-nightly-appimage
```
sed -i "s/pkgver=.*/pkgver=$(date +'%Y.%m.%d')/" PKGBUILD
makepkg --printsrcinfo > .SRCINFO
git add PKGBUILD .SRCINFO
git commit -m "version bump"
```

**Comment by pajlada** [comment, marked as answer]
AppImages no longer supported and we provide `-bin` and `-nightly-bin` packages instead.

Thanks for the work you've put into the PKGBUILDs!
